app/serializers/api/admin/reports/order_serializer.rb

```diff
@@ -0,0 +1,63 @@
+class Api::Admin::Reports::OrderSerializer < ActiveModel::Serializer
```

Comment:
Could you describe at a high level how you're designing this? I don't really understand why we are adding this new set of serializers or how the current reports in `lib/open_food_network` are related to this PR.